test/unit/test_orderlyweb_client_wrapper.py

```diff
@@ -1,18 +1,12 @@
-from src.task_run_diagnostic_reports import run_reports
+from src.utils.run_reports import run_reports
 from src.config import ReportConfig
 from orderlyweb_api import ReportStatusResult
 from unittest.mock import patch, call
 from src.orderlyweb_client_wrapper import OrderlyWebClientWrapper
-from test.unit.test_task_run_diagnostic_reports import MockConfig
+from test.unit.test_run_reports import MockConfig
 
 reports = [ReportConfig("r1", None, ["r1@example.com"], "Subj: r1")]
 
-send_email_call_r1 = call(
```

Comment:
Tests here are basically unchanged but email sending is no longer baked into the `run_reports` method, hence tests use an arbitrary success callback instead.